models/user.py

```diff
@@ -128,6 +129,10 @@ def find_by_role_and_name(cls, role, name):
             & (UserModel.firstName.ilike(likeName) | UserModel.lastName.ilike(likeName))
         ).all()
 
+    @classmethod
+    def find_unassigned_users(cls):
```

Comment:
I think this method name is a bit vague. Unassigned can mean a multitude of things depending on the context, and I have to look at the code in the method to determine what this method should do. Perhaps we can rename this to `pending`? The query `User.pending` returns pending users in contrast to the query `User.find_unassigned_users` returns unassigned users, but what exactly are they not assigned to?

Reply:
I think I would lean toward `pending_authorization` or `without_assigned_role` or something similar as the same thing can probably be said about the method name `pending`